examples/fishjam-chat/app.json

```diff
@@ -49,9 +49,13 @@
         {
           "android": {
             "enableForegroundService": true
+          },
+          "ios": {
+            "enableScreensharing": true
           }
         }
-      ]
+      ],
+      ["./plugin/build/development-pods-plugin.js"] // Only relevant for local development of packages/ios-client
```

Comment:
WDHY about having this plugin in package?
(might not be part of release in future)
This way, if we would create another example app, we can easily reuse it

Reply:
What other project do you mean? Maybe when we have another one let's think about it then? For playground, not releasable projects it's easier to just copy.